content/css/concepts/cursor/cursor.md

```diff
@@ -54,3 +54,4 @@ In the example below, a `pointer` hand will appear whenever the cursor hovers ov
 ```html
 <p style="cursor: pointer;">The cursor is pointing at this paragraph!</p>
 ```
+![cursor-example](https://github.com/Codecademy/docs/assets/148011207/572b61ea-d6c3-42be-b87e-30a670f261c0)
```

Comment:
this is the wrong path it should be -
https://raw.githubusercontent.com/Codecademy/docs/main/media/cursor-example.gif

Please add the file to the media folder and use a filename that explains the content, similar to the path above

Reply:
@caupolicandiaz Thank you for the correction I believe I have done this now let me know if anything else needs to be changed.